src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -98,19 +98,25 @@ object DockerPlugin extends AutoPlugin {
     ),
     dockerUpdateLatest := false,
     dockerAutoremoveMultiStageIntermediateImages := true,
-    dockerLayerGrouping := {
+    dockerLayerGrouping := { _: String =>
+      None
+    },
+    dockerGroupLayers := {
       val dockerBaseDirectory = (defaultLinuxInstallLocation in Docker).value
-      (path: String) =>
-        {
-          val pathInWorkdir = path.stripPrefix(dockerBaseDirectory)
-          if (pathInWorkdir.startsWith(s"/lib/${organization.value}"))
-            Some(2)
-          else if (pathInWorkdir.startsWith("/lib/"))
-            Some(1)
-          else if (pathInWorkdir.startsWith("/bin/"))
-            Some(1)
-          else None
-        }
+      // Ensure this doesn't break even if the JvmPlugin isn't enabled.
+      val artifacts = projectDependencyArtifacts.?.value.getOrElse(Nil).map(_.data).toSet
+      val oldFunction = (dockerLayerGrouping in Docker).value
+
+      // By default we set this to a function that always returns None.
+      val oldPartialFunction = Function.unlift((tuple: (File, String)) => oldFunction(tuple._2))
+
+      val libDir = dockerBaseDirectory + "/lib/"
+      val binDir = dockerBaseDirectory + "/bin/"
```

Comment:
could you also add "/conf/" to the same layer? I noticed that `/conf/application.ini` gets created when `javaOptions` are used.
```
      javaOptions in Universal := Seq(
      "-Djava.net.preferIPv4Stack=true",
      "-Dlogback.configurationFile=logback.packaged.xml"
      ),
```